cocotb/share/include/gpi.h

```diff
@@ -85,7 +85,7 @@ EXTERN_C_START
 typedef enum gpi_event_e {
     SIM_INFO = 0,
     SIM_TEST_FAIL = 1,
-    SIM_FAIL = 2,
+    SIM_FAIL = 2
 } gpi_event_t;
```

Comment:
Any reason for this change? Trailing commas are perfectly legal in C99 and C++11. Perhaps we should close out on #1087 decisively first

Reply:
the commit was removed